### DISCUSSION TITLE
Colour updates?

### DISCUSSION THREAD

**Discussion by Kronrater** [ant-design-blazor/ant-design-blazor#3264] [General]
I apologise if this seems like a silly question, but do you have any idea when Ant Blazor will be updated to match the new Ant Design Colour System as part of their v5.0 release? Specifically, I'm interested in knowing if the text link and button colours will be adjusted to meet the WCAG 2.0 AA coluor contrast requirements (The current ones are quite faint, probably explaining their recent updates). 

I was just curious about this.

![Screenshot 2023-05-17 at 1 37 03 pm](https://github.com/ant-design-blazor/ant-design-blazor/assets/109572486/d1294b51-c24d-4f3b-a449-51c754edf0d4)

**Comment by ElderJames** [comment]
Thank you for contacting us @Kronrater , for now we are using v4.x style yet and we haven't done the v5 style synchronization yet. If you're interested, feel free to contribute!

**Reply by Kronrater** [reply]
Thanks for your quick response, @ElderJames!

Would really love to, but I'm just a pesky Digital Product Designer and will only make things worse there ;-)

Is there a plan to move to v5 styling in the near future or are there other more pressing priorities from your POV?

**Reply by Kronrater** [reply]
**Which would mainly entail the below re: core Design specifications:**

- Basic rounded corner adjustment, changed from 2px to four layers of radius, which are 2px 4px 6px and 8px. For example, radius of default Button is modified from 2px to 6px.
- Primary colour adjustment, changed from #1890ff to #1677ff.

**Reply by ElderJames** [reply]
Hello, we are working on the implementation of css in csharp,and it's a very early stage for v5.

**Reply by xinjiboy1234** [reply]
hello, i am interest about the ant v5, witch branch is the main of v5 version?   there has any roadmap for v5 version?  thank you.

**Reply by ElderJames** [reply]
@xinjiboy1234 The v5 branch is in https://github.com/ant-design-blazor/ant-design-blazor/tree/compatibility-v5

Please see https://github.com/ant-design-blazor/ant-design-blazor/issues/4146
